tests/testthat/test-internal-consistency.R

```diff
@@ -0,0 +1,65 @@
+# setup test --------------------------------------------------------------
+
+read_mun_csv <- function(file) {
+  target_year <- sub(".csv", "", basename(file))
+  df <- read.csv(file) %>%
+    mutate(target_year = target_year)
+
+  return(df)
+}
+
+csv_dir <- system.file("csv/flat", package = "munch")
+all_files <- list.files(csv_dir)
+file_paths <- file.path(csv_dir, all_files)
+
+all_data <-
+  file_paths %>%
+  purrr::map_df(~ read_mun_csv(.)) %>%
+  as_tibble() %>%
+  mutate(target_year = as.integer(target_year))
```

Comment:
Should this be a function in `R/` instead?

Reply:
now an internal function: 8a2bb93